src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -1417,5 +1420,311 @@ internal bool TryGetGuidCore(out Guid value)
             value = default;
             return false;
         }
+
+        internal char GetChar()
```

Comment:
These changes can be reverted once you are invoking the corresponding `JsonConverter<Primitive>`.

Reply:
Removed by force push https://github.com/dotnet/runtime/pull/114863/commits/9047de2e4492706e2d91c31706344d4a6886fe4f